src/oc/web/components/ui/onboard_wrapper.cljs

```diff
@@ -395,26 +395,12 @@
                  :type "text"
                  :auto-capitalize "none"
                  :pattern "@?[a-z0-9.-]+\\.[a-z]{2,4}$"
+                 :value (:email-domain org-editing)
```

Comment:
Using `:value` causes a bug which is why I switched to `(set~ (.-value field)` on `:did-mount`

http://g.recordit.co/IOJ257X59t.gif